src/frontend/src/lib/services/address.services.ts

```diff
@@ -12,20 +12,22 @@ import { assertNonNullish, isNullish } from '@dfinity/utils';
 import { get } from 'svelte/store';
 
 export const loadAddress = async (): Promise<{ success: boolean }> => {
+	const tokenId = ETHEREUM_TOKEN_ID;
+
 	try {
 		const { identity } = get(authStore);
 
 		const address = await getEthAddress(identity);
-		addressStore.set({ address, certified: true });
+		addressStore.set({ tokenId, data: { data: address, certified: true } });
 
 		await saveEthAddressForFutureSignIn({ address, identity });
 	} catch (err: unknown) {
-		addressStore.reset();
+		addressStore.reset(tokenId);
 
 		toastsError({
 			msg: {
 				text: replacePlaceholders(get(i18n).init.error.loading_address, {
-					$symbol: ETHEREUM_NETWORK_SYMBOL
+					$symbol: tokenId.description ?? ''
```

Comment:
We used to use the network symbol here, that was a mistake?

Reply:
I am divided on that actually: basically, we will use that message to tell that there was an error loading the address.

In case of ETH, mainnet and testnet have the same address. For bitcoin no: mainnet address will be different from testnet. So, for consistency either I use the network symbol saying in a generic way that we are referring to the chain. Or we use the tokenId because in a generic way, that is the crypto behind...


Boh.. any opinion here?

Reply:
No strong opinion here, I was more curious to know if the change was expected. Works for me, thanks for the explanation.